prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -567,8 +574,9 @@ class LookupTableCreateCommand : GenericLookupTableCommand(
             return
         }
 
-        // Output the data for review.
-        if (!silent) {
+        // Output the data for review specified and/or the table is small enough.
+        val isLargeTable = inputData.size > 50 || inputData[0].keys.size > 7
+        if (!silent && (showTable || (!showTable && !isLargeTable))) {
```

Comment:
```suggestion
        if (!silent && (showTable || (!showTable && !isLargeTable))) {
```
```
if (!silent && (showTable || !isLargeTable)) {
```
showTable's existence shortcuts to "true"... otherwise you only get a true if !isLargeTable.